restconnector/RestConnector.h

```diff
@@ -11,20 +11,35 @@
 namespace restconnector
 {
 
+/**
+ * The RestConnector class is responsible starting a HTTP server
+ * in a dedicated thread, according to a given hostname and port.
+ * HTTP requests are forwarded to an implicitely registered handler.
+ */
 class RestConnector
 {
 public:
 
-    RestConnector( const std::string& address, const std::string& port );
+    /**
+     * Default constructor.
+     * @param hostname Hostname or IP address
+     * @param port Listening port
+     */
+    RestConnector( const std::string& hostname, const std::string& port );
```

Comment:
OK, ended it up converting the string into uint16_t but did not merge both argument into one. Also agreed with the unit testing being part of the DOD, work in progress.